open_academy/__manifest__.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
```

Comment:
Are you sure that you run pre-commit? This sentence is a lint.

Also remove blank lines in all this file please.